pkg/controller/node/status.go

```diff
@@ -191,7 +195,7 @@ func getUnavailableMachines(currentConfig string, nodes []*corev1.Node) []*corev
 		nodeNotReady := !isNodeReady(node)
 		if dconfig == currentConfig && (dconfig != cconfig || nodeNotReady) {
 			unavail = append(unavail, node)
-			glog.V(2).Infof("Node %s unavailable: different configs %v or node not ready %v", node.Name, dconfig != cconfig, nodeNotReady)
+			glog.V(2).Infof("Node %s unavailable: different configs (desired: %s, current %s) or node not ready %v", node.Name, dconfig, cconfig, nodeNotReady)
```

Comment:
Probably at this point split into separate conditions with separate logging so we know which happened?